gatsby-config.js

```diff
@@ -89,13 +89,14 @@ module.exports = {
     {
       resolve: `gatsby-plugin-manifest`,
       options: {
-        name: `gatsby-starter-default`,
-        short_name: `starter`,
+        name: `Zendesk Garden`,
+        short_name: `Garden`,
+        description: `Garden is a design system for Zendesk where we grow beautifully simple and accessible UI components.`,
```

Comment:
Needs content review. Please add @allisonacs to reviewers.

Reply:
For description we could consider a key-wordy version of the homepage `H2`. 

```
The Garden design system is the source of truth for tools, standards, and best practices when building products at Zendesk.
```

Reply:
I've removed this changeset from the pull request and opted out of `gatsby-plugin-manifest` to bring parity to the current website (no web app manifest).